packages/govern-console/src/utils/types.tsx

```diff
@@ -1,8 +1,8 @@
 export enum CustomTransactionStatus {
-  Successful,
-  Failed,
   Pending,
   InProgress,
+  Successful,
+  Failed,
 }
 
 export enum CiruclarProgressStatus {
```

Comment:
Not sure if it is worth to fix now, but there is a typo in this enum

Reply:
nice catch, thanks